### PR TITLE
Add systemd unit + docs

### PR DESCRIPTION
**Pull request by samveen**
Created a `systemd` unit file to let it handle service startup, logging and crashes/restarts as required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -217,6 +217,17 @@ sudo /home/pi/fbcp-ili9341/build/fbcp-ili9341 &
...
 
+##### Launching the display driver with `systemd`
+
+Alternatively, instead of modifying `/etc/rc.local`, use the provided `systemd` unit file as below:
+
+```bash
+sudo install -m 0644 -t /etc fbcp.conf 
+sudo install -m 0644 -t /etc/systemd/system fbcp.service 
+sudo systemctl daemon-reload
+sudo systemctl enable fbcp && sudo systemctl start fbcp
```
Comment (juj) [inline README.md:228]: Thanks for the PR! Could the name be renamed from `fbcp` to `fbcp-ili9341` to keep with the repository name?

**Comment by juj** [comment]
Some people have observed that modifying /etc/rc.local will initialize the display undesirably late with respect to system startup sequence. Do you happen to know if there is a difference in the startup sequence with systemd vs /etc/rc.local processing, or would it practically be the same?

File: README.md (hunk 1 above)
Comment (samveen) [reply README.md]: done

**Comment by samveen** [comment]
As far as I understand the startup sequence, this `systemd` unit will run BEFORE `rc.local` . That said, I'll have to read up the exact sequencing and get back to you. `systemd` does provide well known targets which can be used to force a particular sequencing if required.
- Does the fbcp-ili9341 require some service to be up and running before it?
- Does it need to be started before some service?  

I can add those requirements into the service unit if you have any info on this. 

The non-sequenced unit  worked for me, which had been [in use](https://github.com/samveen/IdaCam) without issues for more than a year now.

**Comment by juj** [comment]
Ops, something has happened with the PR on README.md, and it shows every line as modified now?

**Comment by samveen** [comment]
I did a `:fileformat unix` in `vim`, as the file was `dos` format (newlines were all `CRLF`). 

Should I undo that change or are you fine with merging it as it?

**Comment by juj** [comment]
Hmm, let's do an undo on that one, since it does make finding commits in history via git blame more difficult. Not sure how the CRLF line endings came about, but probably better to keep them there for now.

**Pull request closed without merging**